inc/class-core-sitemaps-provider.php

```diff
@@ -129,14 +104,12 @@ public function get_url_list( $page_num, $type = '' ) {
 			// Extract the data needed for home URL to add to the array.
 			$url_list[] = array(
 				'loc'     => home_url(),
-				'lastmod' => mysql2date( DATE_W3C, $last_modified[0]->post_modified_gmt, false ),
 			);
 		}
 
 		foreach ( $posts as $post ) {
 			$url_list[] = array(
 				'loc'     => get_permalink( $post ),
-				'lastmod' => mysql2date( DATE_W3C, $post->post_modified_gmt, false ),
```

Comment:
Shouldn't we keep the lastmod for the objects that support modification dates like Posts/Pages/...?

Reply:
What's the benefit of the post object type displaying lastmod, but not any other object type? Especially since it wouldn't be displayed on the sitemap index? I'd opt for consistency here.

Reply:
I assumed we concluded that there is no significant search engine benefit, and that was the primary reason to remove it, in which case even if it's easy the feature is smaller by removing it altogether.

Best evidence I could find is: https://webmasters.stackexchange.com/questions/92507/how-important-is-it-to-include-lastmod-in-a-sitemap